### PR TITLE
`[EXILED::API]` Adding SendFakeSceneLoading

### PR DESCRIPTION
**Pull request by NotZer0Two**
Adds a new way to load scenes to the client from the server without any client mod, what i could document was in the server, its only present "Facility" but by using [AssetRipper](https://github.com/AssetRipper/AssetRipper) there are 6 scenes in total removing the 2 (NewMainMenu and Facility) that we know there are in total other 4 scenes never seen

They are
**MainMenuRemastered**
Seems some rework or the old menu still semi-working

**FastMenu**
Old menu for the server list

**PreLoader**
Is the loading menu

**Loader**
Black menu before the actual main menu

### CODE REVIEW

**Comment by NotZer0Two** [comment]
I've made SceneType for making sure people don't type it wrong or just don't softlock their client if they only want to explore but if in the future some of them are wrong or bugged people can use the first method with string to load or for newer scenes, i will still updated it

File: Exiled.API/Enums/ScenesType.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        /// <summary>
+        /// Its Loading screen at the start of the game.
+        /// ! Will cause crash when trying joining servers !
+        /// </summary>
+        PreLoader,
+
+        /// <summary>
+        /// A black menu before loading the <see cref="NewMainMenu"/>
+        /// </summary>
+        Loader,
```
Comment (ALEXWARELLC) [inline Exiled.API/Enums/ScenesType.cs:46]: English better please.

Comment (NotZer0Two) [reply Exiled.API/Enums/ScenesType.cs]: Probably better than before

**Comment by VALERA771** [comment]
I have doubts about moving methods from `MirrorExtensions` to other classes

**Comment by NotZer0Two** [comment]
Its something already done, i don't know why github imports old one pr i've made https://github.com/Exiled-Team/EXILED/pull/2731

**Review by xNexusACS — CHANGES_REQUESTED** [review]
Fix build conflicts

**Comment by 6hundred9** [comment]
cool

**Pull request merged**